aws/connection_config.go

```diff
@@ -30,6 +32,12 @@ var ConfigSchema = map[string]*schema.Attribute{
 	"session_token": {
 		Type: schema.TypeString,
 	},
+	"max_error_retry": {
```

Comment:
Following AWS' environment variable naming `AWS_MAX_ATTEMPTS`, I think let's change to `max_error_retry_attempts` to match more closely with that and to be more descriptive